account_banking_mandate/models/account_banking_mandate.py

```diff
@@ -64,12 +64,32 @@ def _get_default_partner_bank_id_domain(self):
     payment_line_ids = fields.One2many(
         comodel_name='account.payment.line', inverse_name='mandate_id',
         string="Related Payment Lines")
+    payment_line_ids_count = fields.Integer(
+        compute='_compute_payment_line_ids_count',
+    )
 
     _sql_constraints = [(
         'mandate_ref_company_uniq',
         'unique(unique_mandate_reference, company_id)',
         'A Mandate with the same reference already exists for this company!')]
 
+    @api.multi
+    @api.depends('payment_line_ids')
+    def _compute_payment_line_ids_count(self):
+        for rec in self:
```

Comment:
Please perform this with `read_group` for performance reasons

Reply:
Done